core/chain_makers.go

```diff
@@ -493,7 +493,7 @@ func GenerateVerkleChain(config *params.ChainConfig, parent *types.Block, engine
 		// preState := statedb.Copy()
 
 		// EIP-2935 / 7709
-		blockContext := NewEVMBlockContext(b.header, cm, &b.header.Coinbase)
+		blockContext := NewEVMBlockContext(b.header, cm, cm.Config(), &b.header.Coinbase)
```

Comment:
You're using the member directly everywhere but here. I would prefer to harmonize the usages

Reply:
fixed